File: OCH_Tao/ABC/B/252.js
const input = require("fs").readFileSync("/dev/stdin", "utf8").trim().split("\n");
const A = input[1].split(" ").map(x => parseInt(x, 10));
const B = input[2].split(" ").map(x => parseInt(x, 10));
let X = new Set();
for (const [i, j] of A.entries()) {
  if (j === Math.max(...A)) {
    X.add(i + 1);
  }
}
for (const i of B) {
  if (X.has(i)) {
    console.log("Yes");
    return false;
  }
}
console.log("No");

File: OCH_Tao/ABC/B/252.py
N,K = map(int,input().split())
A = list(map(int,input().split()))
B = list(map(int,input().split()))
X = set([i+1 for i,j in enumerate(A) if j==max(A)])
for i in B:
  if i in X:
    print("Yes")
    break
else:
  print("No")